src/components/Footer.js

```diff
@@ -0,0 +1,18 @@
+
+export default function Footer() {
+    let footerLinkStyling = "flex items-center mx-4"
+
+    return (
+        <footer className="flex ml-auto w-[calc(100vw-250px)]">
+            <div className="flex items-center mx-2 w-[400px] text-sm">
+                © 2023, made with 𓀐 by 🎤 Sing Song, 🫠 Hanny and 🔔 Nel Bell for a better web. Supervisor: Overlord King Kang 🦍
```

Comment:
© 2023, made with 𓀐 by 🎤 Sung Who Sang a Song, 🫠 Hanny Boi and 🔔 Nel Bell for a better web. Supervisor: Ferocious King Kang 🦍